lib_xud/src/core/XUD_HAL.xc

```diff
@@ -43,6 +43,8 @@ unsigned XtlSelFromMhz(unsigned m)
             while(1);
             break;
     }
+
+    return 0b000;
```

Comment:
Fine (though I think putting the return in the default will make the compiler happy)